Docs: Agregar resumen del capitulo 9

IIKUYY opened the pull request:
## what

- Agregar el resumen del capitulo 9 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

IIKUYY committed:
Docs: Agregar resumen del capitulo 9

## what

- Agregar el resumen del capitulo 9 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

semanticmx commented on Ch9/Ejemplo/index.html:
```diff
@@ -0,0 +1,156 @@
+    <meta 
+        charset="UTF-8" />
+    <meta 
+        name="viewport" 
```
este formato está perfecto!

semanticmx commented on Ch9/Ejemplo/index.html:
```diff
@@ -0,0 +1,156 @@
+    <header>
+      <h1>
+        Ejemplo de Formulario
+    </h1>
```
aquí faltó identar el H1 para que esté en línea con la etiqueta de apertura.

semanticmx commented on Ch9/Ejemplo/index.html:
```diff
@@ -0,0 +1,156 @@
+        <input 
+            type="password">
+        <select>
+            <option 
```
cuando solo hay 1 atributo ese si puede ir en la misma línea

semanticmx commented on Ch9/Ejemplo/index.html:
```diff
@@ -0,0 +1,156 @@
+            type="url">
+    </form>
+    <form>
+        <h3>
```
además de etiquetas de headings también como ejemplo puedes ocupar etiquetas label que tienen un atributo especial para los forms que es el `for`

por ejemplo:

```html
<label for=spinbox1>
  Spinbox
</label>
<input 
  id=spinbox1
  type=checkbox>
```

semanticmx commented on Ch9/Ejemplo/index.html:
```diff
@@ -0,0 +1,156 @@
+        <input 
+            type="email"
+            required>
+        <input type="submit" value="submit">
```
este hay que ponerlo en una columna porque son dos atributos

semanticmx reviewed (changes requested):
muy bien, son ajustes mínimos, ya que hay que actualizar el PR puedes agregar un segundo screenshot en un navegador diferente? 

Solo para mostrar que los elementos de los forms se ven de acuerdo a como cada navegador quiera mostrarlos.

IIKUYY committed:
Chores: Corregir archivos de assets

## what

- Agregar screenshot de otro navegador

## why

- Mostrar variedad de ejemplos
## references

-

IIKUYY commented:
Cambios realizados
